constraints.txt

```diff
@@ -0,0 +1,2 @@
+git+git://github.com/david4096/ga4gh-client.git@protobuf31#egg=ga4gh_client
```

Comment:
Thanks @dcolligan ! You're right that we could do that, but I don't think any thing I suggested can't be automated as part of a release. I did add more comments to the files to make it clear how to use them.

Our internal dependencies shouldn't have stuck versions in the `requirements.txt` unless it is a released version. This gives us a way to use git to manage our dependencies in the same way we do with compliance. To pin the version of schemas in the requirements is a misrepresentation of the facts, because if the schemas master moves ahead server/master should break. 

We could do rolling releases to pypi to bump the version number with each accepted PR. Then the requirements.txt would reflect the state of development.